client/src/components/footer/footer.js

```diff
@@ -139,30 +140,23 @@ class Footer extends Component {
                 }
             }
         }
-
-
     }
 
-
     render() {
         const {state, actions} = this.props;
-        let display = state.game.state === "GAME_IDLE" || state.app.body !== 2;
+        let display = (state.game.state === "GAME_IDLE" || state.app.body !== 2) && state.main.lab !== 0;
```

Comment:
This display property is for the whole footer, we want a separate logic for the landing page that deals with just back/next buttons. something like: let backNextDisplay = state.main.lab !== 0; and apply on the back/next footer